docs/index.html

```diff
@@ -32,7 +32,7 @@ <h1><a class="toc-backref" href="#id6">Introduction</a></h1>
 and downloads can be found from <a class="reference external" href="https://pypi.python.org/pypi/robotframework-seleniumlibrary">PyPI</a>.</p>
 <p>SeleniumLibrary works with Selenium 3 and 4. It supports Python 2.7 as well as
 Python 3.6 or newer. In addition to the normal <a class="reference external" href="https://python.org">Python</a> interpreter, it
-works also with <a class="reference external" href="https://pypy.org">PyPy</a> and <a class="reference external" href="https://jython.org/">Jython</a>. Unfortunately <a class="reference external" href="https://www.seleniumhq.org/">Selenium</a> is not
```

Comment:
This is updated in the release process, could you revert the changes.

Reply:
Reverted!